packages/editor/src/store/actions.js

```diff
@@ -347,7 +346,6 @@ export const trashPost =
 		const postType = await registry
 			.resolveSelect( coreStore )
 			.getPostType( postTypeSlug );
-		registry.dispatch( noticesStore ).removeNotice( TRASH_POST_NOTICE_ID );
```

Comment:
There is no need to remove the failed trash notice. If actions fail again, the same notice will be reused. Users will be redirected if they succeed, and WP will display a different notice.

Reply:
Makes sense to me 👍